projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -80,7 +80,7 @@ export default function ProductInterstitial( {
 	const { recordEvent } = useAnalytics();
 	const { onClickGoBack } = useGoBack( { slug } );
 	const { myJetpackCheckoutUri = '' } = getMyJetpackWindowInitialState();
-	const { siteIsRegistering, handleRegisterSite } = useConnection( {
+	const { siteIsRegistering, handleRegisterSite } = useMyJetpackConnection( {
 		skipUserConnection: true,
 		redirectUri: detail.postActivationUrl ? detail.postActivationUrl : null,
```

Comment:
I know it's not belonging to that PR, but what's the point of this condition? 🤔 

`detail.postActivationUrl ? detail.postActivationUrl : null`

if detail.postActivationUrl is nullable, we could probably just pass it along. Or at least simplify the condition to `detail.postActivationUrl || null` if we care for null being passed. What are your thoughts? We also don't have to touch it in this PR, of course!

Reply:
I don't know if it is nullable 🤔 

I'll just go ahead and shorted the ternary 😄